adapter/BUILD

```diff
@@ -27,3 +28,17 @@ go_library(
         "//pkg/adapter:go_default_library",
     ],
 )
+
+inventory_library(
```

Comment:
Sorry forgot to ask one question in my initial review. Do you think it will be simpler if do not require mixer builders (operators) to edit this file and just ask them to edit the main build mixer_gen (does not exists yet) which will take templates and adapters both. Or may be that is your plan, this is a place holder for now.